README.md

```diff
@@ -261,6 +267,9 @@ This repository contains idiomatic C++ client libraries for the following
 * [Cloud Translation API](google/cloud/translate/README.md)
   [[quickstart]](google/cloud/translate/quickstart/README.md)
   [[reference]](https://googleapis.dev/cpp/google-cloud-translate/latest)
+* [Transcoder API](google/cloud/video/README.md)
```

Comment:
Ugh, this is weird because there is also "Live Stream API" and "Video Stitcher API" in `google/cloud/video/`.

I see that `google/cloud/video/README.md` says this is the "Transcoder API C++ Client Library"... I am inclined to break the "Foo API" mold and instead say "Video Services" here and in the README....

What do you think?

Reply:
Done